index.bs

```diff
@@ -2595,6 +2602,10 @@ When this method is invoked, the [=authenticator=] MUST perform the following pr
 
     If |requireUserVerification| is [TRUE], the method of obtaining [=user consent=] MUST include [=user verification=].
 
+        Note: For the overall WebAuthn security properties to hold, the user verified here must be the same user that was
+        verified when this [=authenticator=] created this [=public key credential=] in
+        <a href='#op-makecred-step-user-consent'>Step 6</a> of [[#op-make-cred]].
+
```

Comment:
The "must" here looks normative to me. Should this be in a Note: and not in the algorithm proper?

Reply:
Well, yeah, one could argue that. I made it a note with lowercase 'must' at this point to sidestep the whole breaking change thing.  

That the topic of the Note must be true for webauthn to "work" is tacitly understood by everyone (one hopes) and this is just pedantically stating it. 

in any case I'm happy to make it a normative statement if that's what folks want to do. one way or the other we ought to explicitly state it and get it into the spec.

Reply:
I agree. Let's hear what people think on the 2018-07-09 WG call.